src/lib/hooks/routing/clientSideSmartOrderRouter.ts

```diff
@@ -16,7 +16,7 @@ export function getRouter(chainId: ChainId): AlphaRouter {
 
   const supportedChainId = asSupportedChain(chainId)
   if (supportedChainId) {
-    const provider = RPC_PROVIDERS[supportedChainId]
+    const provider = DEPRECATED_RPC_PROVIDERS[supportedChainId]
```

Comment:
why aren't we switching between these and the non-deprecated based on the feature flag here?